index.html

```diff
@@ -615,15 +619,15 @@ <h2 id="3Defsandabbrevs">Terms, definitions, and abbreviated terms</h2>
       <dd>
         row of <a>pixels</a> within an image or <a>interlaced PNG image</a>.
       </dd>
-        
-      <dt id="3sdr"><dfn>standard dynamic range (SDR)</dfn>
+
+      <dt>standard dynamic range (<dfn>SDR</dfn>)
       </dt>
 
       <dd>
-        an image format capable of storing images with a relatively high dynamic range similar to or in excess of the human visual system's instantaneous dynamic range (~12-14 stops). PNG allows the use of 2 HDR formats, HLG and PQ.
-      </dd>
-      
-        <dt id="3stop"><dfn>stop</dfn>
+        an image format capable of storing images with a relatively low dynamic range of 5-8 <a>stops</a>. Examples include sRGB, Display P3, ITU-R BT.709
+      <p class="note">Standard dynamic range is independent of the primaries and hence, gamut.  Wide color gamur <a>SDR</a> formats are supported by PNG.</p></dd>
```

Comment:
Typo in "Wide color gamur" (you meant "gamut").

Reply:
Ah, thanks. Fixed in https://github.com/w3c/PNG-spec/pull/348/commits/2def8a482042f8b960d75e9056daaa46ae308b3d.